app/views/one_repayment.scala.html

```diff
@@ -48,13 +47,13 @@ <h1 id="main-message">@{LangMessages.`We are processing your VAT repayment`.show
   <td>@{LangMessages.`Amount claimed`.show}</td><td>@{LangMessages.`Estimated repayment date`.show}</td>
  </tr>
  <tr>
-  <td id="amount">£@amountClaimed</td><td id="repay-date">@obligationDates.estimatedPaymentDate</td>
+  <td id="amount">£@onePayment.fold("")(_.amount.toString)</td><td id="repay-date">@onePayment.fold("")(_.estimatedRepaymentDate.toString)</td>
```

Comment:
Hmm, Looks like `onePayment` has to be always there.
I would change method signature to require always onePayment:

```
@(
  onePayment: RepaymentData,  //instead of Option[RepaymentData]
  ...
)
```

And throw exception from controller if `onePayment` isn't there.